[April CQA] CQA fixes for the UI title (terminology, UI formatting, style cleanup) (#361)
migtools/mta-documentation: assemblies/ui-guide/assembly_mta-custom-questionnaire.adoc

Conditional questions::
The YAML syntax supports including or excluding questions based on tags existing on the application or archetype, for example:

> * If the application or archetype has the `Language/Java` tag, the {ProductShortName} includes the `What is the main JAVA framework used in your application?` question in the questionnaire.

Author: ```suggestion
* If the application or archetype has the `Language/Java` tag, the `What is the main JAVA framework used in your application?` question is included in the questionnaire:
```

...
----

> * If the application or archetype has the `Deployment/Serverless` and `Architecture/Monolith` tag, the {ProductShortName} excludes the `Are you currently using any form of container orchestration?` question from the questionnaire:

Author: ```suggestion
* If the application or archetype has the `Deployment/Serverless` and `Architecture/Monolith` tag, the `Are you currently using any form of container orchestration?` question is excluded from the questionnaire:
```

----

Automated answers based on tags present on the assessed application or archetype::

> The {ProductShortName} selects automated answers based on the tags existing on the application or archetype. For example, if an application or archetype has the `Runtime/Quarkus` tag, the {ProductShortName} automatically selects the `Quarkus` answer, and if an application or archetype has the `Runtime/Spring Boot` tag, the {ProductShortName} automatically selects the `Spring Boot` answer:

Author: ```suggestion
The {ProductShortName} selects automated answers based on the tags that exist on the application or archetype. For example, if an application or archetype has the `Runtime/Quarkus` tag, the {ProductShortName} automatically selects the `Quarkus` answer. Similarly, if an application or archetype has the `Runtime/Spring Boot` tag, the {ProductShortName} automatically selects the `Spring Boot` answer:
```

Automatic tagging of applications based on answers::

> If the {ProductShortName} selects this answer during an assessment, it automatically applies tags to the application or archetype. Because these tags are transitive, the {ProductShortName} removes them if you discard the assessment. The following elements define each tag:

Author: ```suggestion
During the assessment, the {ProductShortName} automatically applies tags to the application or archetype based on the answer if this answer is selected.  Because these tags are transitive, the {ProductShortName} removes them if you discard the assessment. The following elements define each tag:
```
